app/res/layout/entity_item_image.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ImageView xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="match_parent" style="@style/EntityItemTextView" android:textAppearance="@style/EntityItemText">
+    android:layout_height="match_parent" style="@style/EntityItemTextView"
+    android:textAppearance="@style/EntityItemText"
+    android:scaleType="centerCrop">

```

Comment:
@wpride The behavior here is "make the image fill the entire space", right? That's not correct and isn't what we want to be doing here. People won't necessarily expect that their icons scale up to fill the entire view, and it will distort smaller images that don't fit in the space.